### ISSUE TITLE
Feature Request: pickchar-like functionality

### ISSUE THREAD

**Issue by metervisaprone** [strongbox-password-safe/Strongbox#719]
There are still websites where I’m required to enter a random selection of characters from a password, passphrase or other secret. There appears to be no easy way to do this in Strongbox (I’d be overjoyed to find out I’m wrong?!), so I have to open the relevant entry, reveal the secret and manually count out the necessary characters (accurately picking characters 13, 17 and 25 from a 30+ character random string is not fun 😁). Keepass has the ‘pickchars’ functionality to support this. Something similar within Strongbox would be great, even if it’s initially just a manual mechanism to enter the numbers and get the correct characters back.
Thanks.

**Comment by strongbox-mark** [comment]
I hear you, this is a bit painful. One thing that can help is the "Large Text View" - Are you aware of this feature? On iOS, long tap on the Password field, or on macOS click the "More" button > Show Large Text View.

But we're hoping to find a better way or improve on this flow.

**Comment by metervisaprone** [comment]
Nope, hadn’t come across the large text view. That should definitely make it easier to do the counting, thanks! But I’m looking forward to the ‘better way’ 😁
